File: loianeTraining/javaBasico/exercicios-aulas-16-17/Ex11.java
import java.util.Scanner;

public class Ex11 {
    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);

        System.out.println("Digite dois numeros");
        var number1 = scan.nextInt();
        var number2 = scan.nextInt();

        var maior = 0;
        var menor = 0;
        if (number1 > number2) {
            maior = number1;
            menor = number2;
        } else {
            maior = number2;
            menor = number1;
        }

        var soma = 0;
        for (int numberAtual = menor; numberAtual <= maior; numberAtual++) {
            soma += numberAtual;
        }
        System.out.println("A soma do intervalo entre os dois numero digitados e: " + soma);
        scan.close();
    }
}

File: Loiane_Training_C#/Basico/D/Ex11.cs
using System;

class Ex11
{
    static void Main(string[] args)
    {
        Console.WriteLine("Digite 2 numeros");
        int number1 = int.Parse(Console.ReadLine());
        int number2 = int.Parse(Console.ReadLine());

        int smallerNumber = number1 < number2 ? number1 : number2;
        int greaterNumber = smallerNumber == number1 ? number2 : number1;

        int sum = 0;
        for (int i = smallerNumber + 1; i < greaterNumber; i++)
        {
            sum += i;
            Console.Write(i + " ");
        }

        Console.WriteLine("Soma: " + sum);
    }
}